src/styles/utils/variables.scss

```diff
@@ -1,4 +1,32 @@
+// Media
```

Comment:
acctually you don't need this comments:

we should understand what does this variable mean by name not for comments